common/test/acceptance/pages/lms/teams.py

```diff
@@ -332,7 +337,8 @@ def __init__(self, browser, course_id, topic):
 
     def is_browser_on_page(self):
         """Check if we're on the create team page for a particular topic."""
-        return self.q(css='.team-edit-fields').present
+        return self.q(css='.team-edit-fields').present \
+            and self.q(css='button.action.action-primary > span.sr').visible
```

Comment:
Nit: don't target the element type, so I'd make this:

    self.q(css='.action-primary > .sr').visible

It might be better to have a semantic class on the span so we can target that.